examples/plugins/src/lighthouse/src/lighthouse.plugin.integration.test.ts

```diff
@@ -89,49 +85,19 @@ describe('lighthouse-create-export-config', () => {
       ]),
     );
   });
-});
-
-describe('lighthouse-create-export-execution', () => {
-  beforeEach(() => {
-    vol.fromJSON(
-      {
-        [LIGHTHOUSE_OUTPUT_FILE_DEFAULT]: JSON.stringify(lhr),
-      },
-      MEMFS_VOLUME,
-    );
-  });
 
-  // TODO: Convert to E2E test or reduce scope, it takes too long to run these tests
-  /* eslint-disable vitest/no-disabled-tests */
-  it.skip('should return PluginConfig that executes correctly', async () => {
-    const pluginConfig = await create({ url: LIGHTHOUSE_URL });
-    await expect(executePlugin(pluginConfig)).resolves.toMatchObject(
-      expect.objectContaining({
-        slug,
-        title: 'Lighthouse',
-        description: 'Chrome lighthouse CLI as code-pushup plugin',
-        duration: expect.any(Number),
-        date: expect.any(String),
-        audits: expect.any(Array),
-        groups: expect.any(Array),
-      }),
-    );
-  });
-
-  it.skip('should use onlyAudits', async () => {
+  it('should use onlyAudits', async () => {
     const pluginConfig = await create({
-      url: LIGHTHOUSE_URL,
+      url: 'http://localhost:8080',
+      outputPath: `${join('tmp', 'lighthouse-report.json')}`,
       onlyAudits: 'largest-contentful-paint',
     });
     expect(pluginConfig.runner.args).toEqual(
       expect.arrayContaining(['--onlyAudits="largest-contentful-paint"']),
     );
-    const { audits: auditOutputs } = await executePlugin(pluginConfig);
-
-    expect(auditOutputs).toHaveLength(1);
-    expect(auditOutputs[0]?.slug).toBe('largest-contentful-paint');
+    expect(pluginConfig.audits).toHaveLength(1);
+    expect(pluginConfig.audits[0]?.slug).toBe('largest-contentful-paint');
```

Comment:
Not formatted correctly. But using `objectContaining`, one can test it in one command 😁 
```suggestion
    expect(pluginConfig).toStrictEqual([expect.objectContaining({
      audits: [expect.object.Containing({ slug: 'largest-contentful-paint'})]
    })])
```